app/assets/stylesheets/finder_frontend.scss

```diff
@@ -365,3 +365,9 @@ mark {
   padding-left: 13px;
   padding-right: 13px;
 }
+
+.app-before-content--inverse {
+  background-color: govuk-colour("blue");
+  padding-top: govuk-spacing(3);
+  overflow: hidden;
```

Comment:
Thanks Martin. I'm not sure if ` @include govuk-clearfix;` would have the desired effect, could you double check on this? I'm seeing the background colour not bleeding full height on a tablet: 
<img width="347" alt="Screenshot 2024-09-11 at 17 01 13" src="https://github.com/user-attachments/assets/ac36db3c-51fc-4596-a2ee-55beed3d4849">

Alternative to this fix would be to amend the contextual breacrumbs component so that classes could be passed to the breadcrumbs component within which could be used to amend the bottom margin of the component.

Reply:
Thanks Hanna, you're right, definitely an issue on tablet.

The fix looks good to me, as you say, ideally we would be able to customise the spacing in the breadcrumbs component which could be a nice improvement to make in the future as a separate piece of work.